src/components/ConnectWallet/Wallets/BitBoxApp.vue

```diff
@@ -0,0 +1,69 @@
+<template>
```

Comment:
There's a lot of code repetition in all of these wallet specific components.

We could get rid of a lot of boilerplate by having connection detail specific components like:

```html
<connection-details>
  <step>
    Make sure Tor is up and running on your system.
    <b-link v-b-modal.tor-modal>Click here</b-link> for instructions.
  </step>
  <step>
    Open BitBoxApp and go to <span class="font-weight-bold">Settings > Enable tor proxy</span>.
  </step>
  <step>
    Check <span class="font-weight-bold">"Enable tor proxy"</span> and enter <copy>127.0.0.1:9050</copy>
  </step>
  ...
</connection-details>
```

Which is equivalent to the current:

```html
<card-widget header="Here's how to connect BitBoxApp to your Umbrel">
    <div class="px-3 px-lg-4 pb-3">
      <ol class="connect-wallet-steps">
        <li class="connect-wallet-step">
          Make sure Tor is up and running on your system.
          <b-link v-b-modal.tor-modal>Click here</b-link> for instructions.
        </li>
        <li class="connect-wallet-step">
          Open BitBoxApp and go to <span class="font-weight-bold">Settings > Enable tor proxy</span>.
        </li>
        <li class="connect-wallet-step">
          Check <span class="font-weight-bold">"Enable tor proxy"</span>
          and enter:
          <input-copy
            value="127.0.0.1:9050"
            size="sm"
            auto-width
          ></input-copy>
        </li>
        ...
      </ol>
    </div>
    <b-modal id="tor-modal" centered hide-footer>
      <tor-setup></tor-setup>
    </b-modal>
</card-widget>
```

Reply:
Done in 6b4f9a7. There are a lot of changes so it might not be clear.

Is it better like this?

Reply:
Loads better!